packages/api/src/hooks/useExchangeRate.ts

```diff
@@ -0,0 +1,32 @@
+import { useCallback } from 'react';
+import useSubscription from '../useSubscription';
+
+type TPayload = Required<
```

Comment:
Making `target_currency` required because there is no implementation of `exchange_rate` in FE that requires us to retrieve all exchange rates. Let me know if need to keep it as optional to maintain flexibility